lib/ship_fosdick/downloader.rb

```diff
@@ -1,27 +1,21 @@
 module ShipFosdick
   class Downloader
-    def initialize(bucket)
-      @bucket = bucket
-    end
 
-    def download
+    def self.download
       [].tap do |returned_content|
         objects.each do |object|
           next unless object.key.downcase.include?('ship')
-          returned_content << object.content
+          next unless object.key.downcase.include?('.txt')
+          returned_content << object.key
         end
       end
     end
 
-    def objects
-      return bucket.objects.with_prefix(ShipFosdick.configuration.folder_prefix) if ShipFosdick.configuration.folder_prefix
-
-      bucket.objects
-    end
-
     private
-
-    attr_reader :bucket
-
+    def self.objects
+      prefix = ShipFosdick.configuration.folder_prefix
+      return ShipFosdick.bucket.objects(prefix: prefix) if prefix
+      ShipFosdick.bucket.objects
```

Comment:
If you call `.bucket.objects(prefix: '')`, is that the same as passing no arguments?